restful_api/myapi/api/resources/quiz_attempt.py

```diff
@@ -48,14 +48,19 @@ class QuizAttemptResource(Resource):
 
     def __init__(self):
         self.schema = QuizAttemptSchema()
+        self.section_completed_schema = SectionCompletedSchema()
+        self.enroll_schema = EnrollSchema()
+        self.quiz_schema = QuizSchema()
 
     def get(self, course_id, section_id, trainer_id, eng_id):
         try:
             query = (
                 QuizAttempt.query
-                .join(Quiz, (QuizAttempt.section_id == Quiz.section_id) & (QuizAttempt.course_id == Quiz.course_id), isouter=True)
                 .join(Employee, isouter=True)
                 .filter(Employee.id == eng_id)
+                .filter(QuizAttempt.course_id == course_id)
+                .filter(QuizAttempt.section_id == section_id)
+                .filter(QuizAttempt.trainer_id == trainer_id)
```

Comment:
```
.filter((QuizAttempt.course_id == course_id) & (QuizAttempt.section_id == section_id) & (QuizAttempt.trainer_id == trainer_id))
```

This should work but not important, up to u

Reply:
oh ya HAHA forgot I could do it that way